HackBeanpot/application-portal
# Export to CSV

closes #93 

you can now download a CSV of the exported applicants data, which respects sort and filter.

## to test

1. visit the preview link
2. go to the admin tab (make sure logged in w hbp email)
3. click `view / modify applicants`
4. apply a sort
5. apply a filter
6. click `export all data`
